### ISSUE TITLE
Update the Readme to include Postgres setup

### ISSUE THREAD

**Issue by mathyoudawson** [magara/magara#28]
While following the installation guide, the setup script aborted as I didn't have a Postgres user set up with the appropriate permissions. 

If the readme had a quick walkthrough, explaining how to set up a Postgres user with the correct permission, it may help new contributors.

**Comment by mathyoudawson** [comment]
Fixed in 74dc344

**Comment by mathyoudawson** [comment]
The set up script could also be changed to check if a Postgres user has been created with the appropriate permissions, and if not, create said user. @BerkhanBerkdemir Would be keen to hear your opinion on this. I would be happy to edit the set up script but unsure if it's worth my time.

**Comment by BerkhanBerkdemir** [comment]
Yes, you are right, @mathyoudawson. However, I already provide `docker-compose.yml` for this purpose which new contributors don't need to worry about setting a PostgreSQL.

**Comment by mathyoudawson** [comment]
The Developer Installation Guide looks great. We should add different OS's though? Looks like another issue has been raised where someone can't install

**Comment by BerkhanBerkdemir** [comment]
Issue #34 covers that, @mathyoudawson.
